src/components/DataViewer/DataViewer.tsx

```diff
@@ -9,23 +9,28 @@ export interface Data {
 }
 
 const Key = styled.div`
+    // TODO VFB-16 Change to using theme with palettes of colours (dark font matching with this background)
     color: ${(props) => props.theme.foregroundColor};
+    // TODO VFB-16 Change to using theme with palettes of colours
     background-color: ${(props) => props.theme.secondaryBackgroundColor}40;
     display: inline-block;
-    font-size: medium;
     border-radius: 0.7em;
     padding: 0.2em 0.5em;
 `;
 
 const Value = styled.div`
-    font-size: medium;
     padding: 0.2em 0.5em;
 `;
 
 const EachItem = styled.div`
     padding-bottom: 1em;
 `;
 
+const ContentDiv = styled.div`
+    width: 1000px;
+    max-width: 100%;
+`;
```

Comment:
use `width: min(100%, 1000px)`

Reply:
Hmmm it is not working - so maybe we will stick to the original one